File: client/public/Codes/Python/FibonacciSequence.py
nterms = int(input("How many terms? "))
n1, n2 = 0, 1
count = 0


if nterms <= 0:
    print("Please enter a positive integer")
elif nterms == 1:
    print("Fibonacci sequence upto", nterms, ":")
    print(n1)
else:
    print("Fibonacci sequence:")
    while nterms > count:

        n3 = n1 + n2
        n1 = n2
        n2 = n3
        print(n1)
        count += 1

File: client/public/Codes/C/FibonacciSequence.c
#include <stdio.h>

int main()
{
   int a, b, n, next;

   a = 0;
   b = 1;
   next = a + b;

   printf("Enter Number of Terms: ");
   scanf("%d", &n);

   printf("Fibonacci Series : %d %d ", a, b);

   for (int i = 3; i <= n; ++i)
   {
      printf("%d ", next);
      a = b;
      b = next;
      next = a + b;
   }

   return 0;
}